main.js

```diff
@@ -564,6 +571,23 @@ app.on('ready', function() {
       require('open')('mailto:project@openbazaar.org?subject=OpenBazaar Debug Report&body=' + body);
 
     }},
+    {label: 'View Python Error Log', type: 'normal', click: function() {
```

Comment:
If the python error log isn't created, the View Python Error Log menu option throws and error. Can this be changed to either only add the option if the file exists, or handle the error if it doesn't?